docs/source/concepts.rst

```diff
@@ -24,3 +24,5 @@ so it should be no problem to load and process for example time series.
 
 If you want to process data with, for example, 1D or 3D samples, you can write
 :ref:`UDFs <user-defined functions>`. Note that in that case, a "frame" is no longer 2D!
+
+MATLAB users should keep one thing in mind. The navigation axes in Python is the transpose of that of MATLAB. In Python, the indices increase linearly with the row. ``[[1,2,3],[4,5,6],[7,8,9]]`` is how a 3x3 matrix is represented. 
```

Comment:
Maybe should be moved below the "numpy convention for axes order" paragraph, as it is directly related.

The `3x3` example should be expanded on a bit - it could be misunderstood as showing the flattened indices of the matrix (MATLAB starts indexing at 1, IIRC!). Or, maybe instead of the last sentence, there could be a link to https://numpy.org/doc/1.18/user/numpy-for-matlab-users.html#numpy-for-matlab-users here?